api/src/services/meetingsService.ts

```diff
@@ -104,3 +104,14 @@ export const createMeetingNote = async (
   });
   return { id };
 };
+
+export const participantExists = async (
+  meetingId: number,
+  principalId: number
+) => {
+  const [participant] = await db('participants').select('principal_id').where({
```

Comment:
I don't understand why this selects `principal_id`. Is there a more intuitive thing this query could select?

Reply:
meetingID for instance ?

Reply:
I recall we had set it to just select `1`. Did that not work? Why not just select the `id` column?

Reply:
i replaced principal_Id with meeting_Id

Reply:
> I recall we had set it to just select `1`. Did that not work? Why not just select the `id` column?

it did i just think querying meeting_id is more self-explanatory than just adding id or 1.

Reply:
Querying for `1` makes it obvious that it's not querying for data from the table. By querying for `meeting_id` but then not using it, it's confusing why the data is being fetched at all. Querying for `id` at least says, "Just trying to figure out if this row exists."

Separately, some databases have an `EXISTS` keyword for this purpose. In ORMs, exists queries are often translated to `SELECT 1 FROM...` because it works across databases.

Does Knex have an `EXISTS` feature? If not, it makes sense to follow known conventions and to not include code that doesn't do anything.

Reply:
Ok, now I understand.